repositorytools/cli/common.py

```diff
@@ -37,12 +37,15 @@ def _get_parser(self):
 
     def __init__(self):
         self.parser = self._get_parser()
-        self.parser.add_argument("-D", "--debug", action="store_true", dest="debug", default=False,
-                                 help="Print lots of debugging information")
+        self.parser.add_argument("-D", "--debug", action="store_true", help="Print lots of debugging information")
+        self.parser.add_argument("-V", action="store_true", dest="display_version", help="Prints version and exit")
         self.repository = None
 
     def run(self, args=None):
         args_namespace = self.parser.parse_args(args)
+        if args_namespace.display_version:
```

Comment:
If you don't like this behaviour, feel free to ignore this PR, and potentially include this feature by yourself later on.
This commit was only planned as a minor improvement, I don't think it's worth a convoluted workaround :)